src/ert/gui/simulation/simulation_panel.py

```diff
@@ -217,3 +220,5 @@ def toggleSimulationMode(self):
     def validationStatusChanged(self):
         widget = self._simulation_widgets[self.getCurrentSimulationModel()]
         self.run_button.setEnabled(widget.isConfigurationValid())
+        if self.run_button.text() == EXPERIMENT_IS_RUNNING_BUTTON_MESSAGE:
```

Comment:
Would it make sense to setEnabled() only once checking both conditions? You can see which you like best.